rc/extra/autorestore.kak

```diff
@@ -41,19 +52,19 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
 ## Remove all the backups that have been created for the current buffer
 def autorestore-purge-backups -docstring "Remove all the backups of the current buffer" %{
-    nop %sh{
-        if [ ! -f "${kak_buffile}" ]; then
-            exit
-        fi
+    %sh{
+        [ ! -f "${kak_buffile}" ] && exit
 
         buffer_basename="${kak_bufname##*/}"
         buffer_dirname=$(dirname "${kak_bufname}")
 
-        ls -1 "${buffer_dirname}"/."${buffer_basename}".kak.* 2>/dev/null | while read -r f; do
-            rm -f "${f}"
-        done
+        rm -f "${buffer_dirname}/.${buffer_basename}.kak."*
+
+        printf %s\\n "
+          echo -markup {Information}Backup files removed
+          echo -debug Backup files removed with exit status: $? 
```

Comment:
I think this is a remaining debug line that should not be in the commited version.